content/articles/multi-output-classification-with-machine-learning/index.md

```diff
@@ -0,0 +1,303 @@
+
+![Dataset image](/engineering-education/multi-output-classification-with-machine-learning/dataset-image.png)
+
+From the image above, our dataset has four columns: `title`, `description`, `type`, and `rating`. The `title` column will be the input column while type` and `rating` will be the output column. We now need to load this dataset on our machine.
+
+To download this Netflix dataset, click [here](https://drive.google.com/file/d/1524vMr5iCGEEJmPx2n2-TAl8EicKuxcg/view?usp=sharing)
+
+### Loading the dataset
+There are various exploratory data analysis (EDA) packages that will load our dataset. Let's import them using the following code:
+
+```python
+import pandas as pd
+import numpy as np
+```
+We will use Pandas to load the dataset. Numpy will be used to perform computational operations on our dataset. It also works well with arrays.
+
+Let's now load the Netflix dataset that you have downloaded from the link above.
+
+```python
+df = pd.read_csv("netflix_titles_dataset.csv")
+```
+To check if our dataset is loaded successfully, run the code:
+
+```python
+df.head()
+```
+This command will output the structure of our dataset, it shows all the columns on our dataset. It should have the same structure as the dataset you have downloaded. The output is shown below:
+
+![Loaded dataset](/engineering-education/multi-output-classification-with-machine-learning/loaded-dataset.png)
+
+Now that we have loaded our dataset successfully, let's now check the distribution of our target/output columns.
+
+### Output columns distribution
+From our dataset, we have two output columns: `type` and `rating`. Column distribution is the value count of each column in the entire dataset. We will start with the `type` column.
+
+#### `type` column
+```python 
+df['type'].value_counts()
+```
+The output is shown below:
+
+```bash
+Movie      4788
+TV Show    2143
+Name: type, dtype: int64
+```
+In the output above, we have 4788 `movie` data samples and 2143 `TV Show` data samples.
+
+#### `rating` column
+To get the value count of the `rating` column, use the following code:
+
+```python
+df['rating'].value_counts()
+```
+The output is shown below:
+
+```bash
+TV-MA    2863
+TV-14    1931
+TV-PG     806
+R         665
+PG-13     386
+TV-Y      280
+Name: rating, dtype: int64
+```
+The output above shows the distribution of all the ratings in our dataset. We have seven ratings: `TV-MA`, `TV-14`, `TV-PG`, `R`, `PG-13` and `TV-Y`.
+
+Before we start building our model, we also need to clean our dataset. Dataset cleaning involves correctly formatting our dataset.
+
+### Text cleaning
+For text cleaning, we will convert all our text data into lower case and also remove stop words. We will use the NeatText Python package to perform this process. We will install Neattext using the following code:
+
+```bash
+! pip install neattext
+```
+Let's import the Neattext functions that will be used for text cleaning.
```

Comment:
This is another case of using passive voice. Instead of using "will be used...", consider replacing that with "...will use"